src/cloudkit.cs

```diff
@@ -977,17 +1278,72 @@ interface CKReference : NSSecureCoding, NSCopying {
 		[Export ("initWithRecord:action:")]
 		IntPtr Constructor (CKRecord record, CKReferenceAction action);
 
-		[Export ("referenceAction", ArgumentSemantic.UnsafeUnretained)]
+		[Export ("referenceAction", ArgumentSemantic.Assign)]
 		CKReferenceAction ReferenceAction { get; }
 
 		[Export ("recordID", ArgumentSemantic.Copy)]
 		CKRecordID RecordId { get; }
 	}
 
+	[iOS (10,0), Watch (3,0), TV (10,0), Mac (10,12, onlyOn64 : true)]
+	[BaseType (typeof(CKSubscription))]
```

Comment:
add `[DisableDefaultCtor]` since there's a designated initializer that does not accept `null` for all it's parameters